prow/pubsub/subscriber/subscriber.go

```diff
@@ -43,6 +45,22 @@ const (
 	postsubmitProwJobEvent = "prow.k8s.io/pubsub.PostsubmitProwJobEvent"
 )
 
+// Ensure interface is intact
+var _ prowCfgClient = (*config.Config)(nil)
+
+type githubClient interface {
+	GetRef(org, repo, ref string) (string, error)
+	// Compatible with config.githubClient
+	GetPullRequestChanges(org, repo string, number int) ([]github.PullRequestChange, error)
```

Comment:
We should try to avoid using a GitHub client, we don't want to assume that the job uses GitHub for source control.

This function is also unused.

Reply:
good point, deleted